### DISCUSSION TITLE
How to ssh into box with no HDMI support?

### DISCUSSION THREAD

**Discussion by denisidoro** [devmfc/debian-on-amlogic#26] [Q&A] [answered]
I have a S905w2 box with no support for HDMI (see https://github.com/devmfc/debian-on-amlogic/discussions/20)

I want to use it as a headless server, so that's not a blocker.

But how can I run commands from it? Is the ssh server started on boot? If so, what's its password?

**Comment by devmfc** [comment, marked as answer]
Yes, of course the ssh server is started at boot. Username and password are the same as for console login.

If you have a H96MaxW2 you could wait a couple of days, because the next release will add support for that box.

**Comment by denisidoro** [comment]
That's awesome news! Thanks
